backend/app/api/v1/endpoints/quay/quayJobs.py

```diff
@@ -13,7 +13,7 @@
     "/api/v1/quay/jobs",
     summary="Returns a job list",
     description="Returns a list of jobs in the specified dates of requested size. \
-            If not dates are provided the API will default the values. \
+            If not dates are provided the API will use the following values as defaults. \
```

Comment:
"If dates are not provided"

Reply:
It may also be clearer if it ended in a colon `:`.